lib/web_ui/dev/test_runner.dart

```diff
@@ -272,15 +272,20 @@ class TestCommand extends Command<bool> {
   }
 }
 
-void _copyAhemFontIntoWebUi() {
-  final io.File sourceAhemTtf = io.File(path.join(
-      environment.flutterDirectory.path,
-      'third_party',
-      'txt',
-      'third_party',
-      'fonts',
-      'ahem.ttf'));
-  final String destinationAhemTtfPath =
-      path.join(environment.webUiRootDir.path, 'lib', 'assets', 'ahem.ttf');
-  sourceAhemTtf.copySync(destinationAhemTtfPath);
+const List<String> _kTestFonts = <String>['ahem.ttf', 'Roboto-Light.ttf'];
```

Comment:
Roboto Light is quite thin, so this has a small risk of having extra antialias problems (but it'll be a cool test for the fuzzy comparison!)

Reply:
For some reason I thought light was the default. Strangely the tests passed locally. I would expect them to fail if I picked a wrong font. Trying `Roboto-Regular`.

Reply:
Of course, it would help if the smoke test used the Flutter engine at all! LOL. Fixing.